errata/admin.py

```diff
@@ -2,4 +2,61 @@
 
 from django.contrib import admin
 
-# Register your models here.
+from .models import (
+    Errata,
+    Type,
+    Status,
+    Log,
+    AreaAssignment,
+)
+
+
+class ErrataAdmin(admin.ModelAdmin):
+    search_fields = ["rfc_to_be__rfc_number"]
+    list_display = [
+        "rfc_to_be",
+        "verifier_dt_person",
+        "status",
+        "type",
+        "submitter_email",
+        "submitted_at",
+    ]
```

Comment:
Should probably add `rfc_to_be` and `verifier_dt_person` to `raw_id_fields` to keep the UI usable